assets/js/vue/google-map.js

```diff
@@ -0,0 +1,140 @@
+  data() {
+    return {
+      city: '',
+      street: '',
+      state: '',
+    }
+  },
+  computed: {
+    classList() {
+      return ['position-relative mx-3 mx-md-8 vue-component']
+    },
+    contactVal() {
+      return Tools.getJSON(this.contact);
+    },
+    items(){
+      const objects = [
+        {
+          margin: 'mb-5',
+          icon: 'fa-phone',
+          truncate: '',
+          text: 'Call us',
+          info: this.contactVal.phone
+        },
+        {
+          margin: 'mb-5',
+          icon: 'fa-envelope',
+          truncate: 'text-truncate',
+          text: 'Email us',
+          info: this.contactVal.email
+        },
+        {
+          margin: '',
+          icon: 'fa-map-marker-alt',
+          truncate: '',
+          text: 'Visit us',
+          info: this.contactVal.street ? this.contactVal.street : this.street
+        }
+      ]
+      return objects
+    },
+    leafletOptions(){
+      const coordinates = this.coordinates.split(',')
+
+      const street = Tools.isTrue(this.contactVal.street) ? this.contactVal.street : this.street
+      const state = this.state
+
+      const obj = {
+        map: {
+          scrollWheelZoom: false,
+          coords: coordinates,
+        },
+        marker: [
+          {
+            coords: coordinates,
+            icon: {
+              iconUrl: 'https://front.trustedroot.net/dist/assets/svg/components/map-pin.svg',
+              iconSize: [50, 45]
+            },
+            popup: {
+              text: `${street}, ${state}`,
+              title: 'Address'
+            }
+          }
+        ]
+      }
+
+      return JSON.stringify(obj);
+    }
+  },
+  mounted() {
+    this.loadMap()
+  },
+  methods: {
+    async setAddress(latitude, longitude){
+      const apiUrl = `https://nominatim.openstreetmap.org/reverse?format=jsonv2&lat=${latitude}&lon=${longitude}`;
+      const response = await fetch(apiUrl)
+      const data = await response.json()
+      const address = data.address;
+      this.city = address.city;
+      this.state = address.state;
+      this.street = address.road;
+
+    },
+    async loadMap(){
+      const mapRef = this.$refs.map;
+      const LeafletLib = $?.HSCore?.components?.HSLeaflet;
+
+      if (LeafletLib === null || L === null) return;
+      LeafletLib.init(mapRef);
+
+      const coordinates = this.coordinates.split(',')
+      await this.setAddress(coordinates[0], coordinates[1])
+    },
+  },
+  props: {
+    coordinates: {
+      default: null,
+    },
+    contact: {
+      default: null
+    },
+  },
+  template: `
+  <div id="contactsSection" :class="classList">
+    <div class="container space-1 space-lg-3">
+      <div class="row justify-content-md-end">
+        <div class="col-md-6 col-lg-5">
+          <div class="card bg-white position-relative z-index-999 p-5 p-sm-7">
+
+            <div class="mb-5">
+              <span class="d-block font-size-2 text-dark text-lh-sm">{{ this.city }},</span>
```

Comment:
In der statischen Version steht hier "Büro", wieso ist das jetzt City und wird aus der Schnittstelle gezogen? this.state gibt es als Daten in der alten Version auch nicht. Die Inhalte sollten sich nicht austauschen.